daemon/cmd/daemon_main.go

```diff
@@ -1755,6 +1755,25 @@ func startDaemon(d *Daemon, restoredEndpoints *endpointRestoreState, cleaner *da
 		}
 	}
 
+	if option.Config.EnableEnvoyConfig {
+		if !d.endpointManager.IngressEndpointExists() {
+			// Creating Ingress Endpoint depends on the Ingress IPs having been
+			// allocated first. This happens earlier in the agent bootstrap.
+			if (option.Config.EnableIPv4 && len(node.GetIngressIPv4()) == 0) ||
+				(option.Config.EnableIPv6 && len(node.GetIngressIPv6()) == 0) {
+				log.Info("Ingress IPs are not available, skipping creation of the Inress Endpoint")
```

Comment:
As a user this info message is not useful to me. Ok so "Ingress endpoint" is not created, now what? Will Cilium function properly? If no, then why don't we error / fatal instead? If Cilium will function properly then why is it important to create an Ingress Endpoint?

Reply:
@aanm Changed this to:
```
				log.Warn("Ingress IPs are not available, skipping creation of the Ingress Endpoint: Policy enforcement on Cilium Ingress will not work as expected.")
```

Does this resolve your concern?

Reply:
This discussion was completed in Slack after the fact. In summary, there is no policy breach if the Ingress Endpoint is not created, but Cilium Ingress traffic will be dropped if that is the case.